lib/customer_io_config.dart

```diff
@@ -0,0 +1,47 @@
+import 'customer_io_enums.dart';
+
+/// Configure plugin using CustomerIOConfig
+class CustomerIOConfig {
+  final String siteId;
+  final String apiKey;
```

Comment:
Can we move these 2 properties out of the config object and into parameters of Dart's `CustomerIO.initialize()` function? To match the native SDKs?